release_notes/osd_latest_product_updates.adoc

```diff
@@ -45,12 +45,12 @@ administrators can now do so by setting the `requests.storage` value for user's
 projects:
 
 ----
-$ oc create quota my-quota \
+$ oc create quota my-quota -n my-project \
     --hard=requests.storage=10Gi,persistentvolumeclaims=50
 
 $ oc describe quota
 Name:                     my-quota
-Namespace:                default
+Namespace:                my-project
```

Comment:
@spurtell I added a `-n` and a non-`default` project here, since that seemed more appropriate for an OSD admin.